pynucastro/constants/constants.py

```diff
@@ -20,10 +22,14 @@
 
 # Boltzmann constant in erg/K and MeV/K
 k = physical_constants.value("Boltzmann constant") / physical_constants.erg
-k_MeV = physical_constants.value("Boltzmann constant in eV/K") / physical_constants.mega
+# SciPy 1.15.0 fixed how exact values are calculated, which makes them slightly
+# more accurate compared to 1.14.1 and earlier. Calculate this one manually for
+# consistency between versions (should match 1.15.0).
+k_MeV = physical_constants.value("Boltzmann constant") / physical_constants.e / physical_constants.mega
```

Comment:
why is this `physical_constants.e`?  isn't that the electric charge?

Reply:
It's from https://github.com/scipy/scipy/blob/1a6c320d36da1e63739c45249172b8e5316c19aa/scipy/constants/_codata.py#L1564.  It works out since `1 eV = 1 V * q_e = 1 J/C * 1.602e-19 C = 1.602e-19 J`.

Reply:
ah indeed -- can you just add a comment saying that?